src/mame/drivers/pentagon.cpp

```diff
@@ -217,8 +217,11 @@ void pentagon_state::machine_reset()
 
 void pentagon_state::video_start()
 {
-	m_frame_invert_count = 16;
-	m_screen_location = m_ram->pointer() + (5 << 14);
+	spectrum_128_state::video_start();
+	m_contention_pattern = {};
+	/* XXX Must be 4 but that requires draw of previous border chunk and significantly complicates the code.
+	Will see if we really have case for this change. */
```

Comment:
Please use `TODO` or `FIXME` to indicate severity of things like this (they’re also picked up by editors like vim and several static analysis tools), not `XXX`.